build.gradle

```diff
@@ -0,0 +1,52 @@
+plugins {
+    id 'java'
+    id "com.diffplug.gradle.spotless" version "3.27.0"
+}
+
+group 'com.adscore'
+version '1.0.4' //version of samples match with the version of the client-libs-java project
```

Comment:
I dont think we should 'bound' version of sample to version of library, the bound is represented in dependency.

Reply:
Okay, I've set it to 1.0.0.

Reply:
:+1: